recipes/structuregraph-helpers/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "structuregraph-helpers" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/structuregraph_helpers-{{ version }}.tar.gz
+  sha256: 8b21a4cf58de1920715ee4ffaf2933ac0a808b7a495382ec9e7020a82b7b74f6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

```suggestion
    - python >=3.8
```
Ref: https://github.com/kjappelbaum/structuregraph-helpers/blob/1d84458490a0be8d6685400283a5fe9dc7a0d0bc/setup.cfg#L58

---

thanks, It is actually an issue upstream in `setup.cfg` that I updated there now.